src/include/common/string_utils.h

```diff
@@ -42,6 +42,8 @@ class StringUtils {
         return c == ' ' || c == '\t' || c == '\n' || c == '\v' || c == '\f' || c == '\r';
     }
 
+    static bool CharacterIsNewline(char c) { return c == '\n' || c == '\r'; }
```

Comment:
I'd make sure a similar function does not exist anywhere else in our code base. This looks like a quite basic function.